.github/workflows/build-test-core-x86-ocaml5.yaml

```diff
@@ -0,0 +1,45 @@
+# coupling: This is a copy of build-test-core-x86.yaml but for OCaml 5.
```

Comment:
@IagoAbal note that this workflow was never run. It's because you didn't include it in tests.yml like
the other workflow.
But don't worry, I've fixed it (and factorized things) in https://github.com/returntocorp/semgrep/pull/8943